op-deployer/pkg/deployer/inspect/genesis.go

```diff
@@ -39,7 +44,7 @@ func GenesisCLI(cliCtx *cli.Context) error {
 	return nil
 }
 
-func GenesisAndRollup(globalState *state.State, chainID common.Hash) (*core.Genesis, *rollup.Config, error) {
+func GenesisAndRollup(lgr log.Logger, globalState *state.State, chainID common.Hash) (*core.Genesis, *rollup.Config, error) {
```

Comment:
why does this need a logger now? it seems like a superfluous argument

Reply:
Whoops, that was brought in in the first commit. Forgot to clean it up.
Done now.